md/resource-management.md

```diff
@@ -75,6 +75,10 @@ If you have specified a resource permission that is not defined in the [REST API
 
 After a resource is added to the portal, it can be used in an [application](applications.md) or in a custom profile.
 
+::: warning
+**Warning**: As the deployment of resource automatically create the appropriate resource permission mappings, be careful to not [push configuration](BonitaBPM_platform_setup.md#update_platform_conf) without doing a pull previously. This would lost the running configuration.
```

Comment:
```suggestion
**Warning**: As the deployment of resources automatically creates the appropriate resource permissions mapping, be careful not to [push a configuration](BonitaBPM_platform_setup.md#update_platform_conf) that has been updated before the deployment. This would overwrite the deployed configuration. To avoid doing so, pull the latest configuration, edit the needed elements and push it back to the Bonita runtime in a pretty short amount of time.
```